pages/configuration/https.rst

```diff
@@ -44,7 +44,7 @@ Create PKCS#5 and X.509 certificate::
   Country Name (2 letter code) [AU]:DE
   State or Province Name (full name) [Some-State]:Hamburg
   Locality Name (eg, city) []:Hamburg
-  Organization Name (eg, company) [Internet Widgits Pty Ltd]:Graylog, Inc.
+  Organization Name (eg, company) [Internet Widgets Pty Ltd]:Graylog, Inc.
```

Comment:
This is actually "correct", as it's the default in OpenSSL:
https://github.com/openssl/openssl/blob/OpenSSL_1_1_0d/apps/openssl.cnf#L139